.github/workflows/test-deps.yml

```diff
@@ -61,15 +62,11 @@ jobs:
                 python-version: ${{ matrix.python-version }}
 
             - name: Install developer dependencies
-              run: python -m pip install requirements/requirements-dev.in
-
-            # TODO: vendor only the parts that we need from the following action
-            - name: Docker
-              if: ${{ matrix.os == 'macos-latest' }}
-              uses: douglascamata/setup-docker-macos-action@5f13b5b88a09467c10f8462810902b60ff70fb75
+              run: |
+                python3 -m pip install --upgrade pip-tools
+                python3 -m pip install .[dev]
```

Comment:
I'm using the command `python3 -m pip install -e .[dev]`, and `-e` means editable, which this has no need for, so this seems to be right (just checking).